Formula/monkeysphere.rb

```diff
@@ -20,10 +21,14 @@ class Monkeysphere < Formula
   end
 
   depends_on "gnu-sed" => :build
+  depends_on "bash" # Apple's BASH 3.2 is insufficient, BASH 4.X features are used
+  depends_on "coreutils"
+  depends_on "findutils" # GNU extensions are used with find
```

Comment:
Only during the build right?

Reply:
No, these are installed along with the monkeysphere formula on user's machines.